pkg/controller/controllercmd/cmd.go

```diff
@@ -39,6 +39,7 @@ type ControllerCommandConfig struct {
 	version       version.Info
 
 	basicFlags *ControllerFlags
+	logs       *logs.Options
```

Comment:
is this a mandatory change? why do we have to change it?

Reply:
yes, it is. klog global flags are removed in 1.23 (https://github.com/kubernetes/enhancements/issues/2845) and need to be registered so we don't get these: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/1269/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-aws-operator-disruptive-single-node/1468226993714630656

Reply:
posted a small explanation here: https://github.com/openshift/cluster-kube-controller-manager-operator/pull/581#pullrequestreview-826829435

Reply:
fortunately took a bad turn and it is not mandatory change but an optional one :)